string obj_id
bool gripper_state
---
bool status